docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc

```diff
@@ -3499,6 +3499,152 @@ CREATE TABLE t1 (c1 int not null primary key, c2 char(50));
+  , CA_LOCATION_TYPE                 string
+  )
+  stored as orc
+  tblproperties
+ (
+    "orc.bloom.filter.columns"="CA_COUNTY, CA_CITY, CA_GMT_OFFSET, CA_LOCATION_TYPE"
+   ,"orc.stripe.size"="67108864"
+  )
+;
+
+
+/* Trafodion DDL */
+
+REGISTER /*INTERNAL*/ HIVE TABLE HIVE.TPCDS_SF1000_ORC.customer_address1;
+/* ObjectUID = 8789483533085275772 */
+
+CREATE EXTERNAL TABLE customer_address1
+  (
+    CA_ADDRESS_SK                    INT DEFAULT NULL NOT SERIALIZED
+  , CA_ADDRESS_ID                    VARCHAR(16) CHARACTER SET ISO88591 COLLATE
+      DEFAULT DEFAULT NULL NOT SERIALIZED
+  , CA_STREET_NUMBER                 VARCHAR(10) CHARACTER SET ISO88591 COLLATE
+      DEFAULT DEFAULT NULL NOT SERIALIZED
+  , CA_STREET_NAME                   VARCHAR(60) CHARACTER SET ISO88591 COLLATE
+      DEFAULT DEFAULT NULL NOT SERIALIZED
+  , CA_STREET_TYPE                   VARCHAR(15) CHARACTER SET ISO88591 COLLATE
+      DEFAULT DEFAULT NULL NOT SERIALIZED
+  , CA_SUITE_NUMBER                  VARCHAR(10) CHARACTER SET ISO88591 COLLATE
+      DEFAULT DEFAULT NULL NOT SERIALIZED
+  , CA_CITY                          VARCHAR(60) CHARACTER SET ISO88591 COLLATE
+      DEFAULT DEFAULT NULL NOT SERIALIZED
+  , CA_COUNTY                        VARCHAR(30) CHARACTER SET ISO88591 COLLATE
+      DEFAULT DEFAULT NULL NOT SERIALIZED
+  , CA_STATE                         VARCHAR(2) CHARACTER SET ISO88591 COLLATE
+      DEFAULT DEFAULT NULL NOT SERIALIZED
+  , CA_ZIP                           VARCHAR(10) CHARACTER SET ISO88591 COLLATE
+      DEFAULT DEFAULT NULL NOT SERIALIZED
+  , CA_COUNTRY                       VARCHAR(20 CHARS) CHARACTER SET UTF8
+      COLLATE DEFAULT DEFAULT NULL NOT SERIALIZED
+  , CA_GMT_OFFSET                    REAL DEFAULT NULL NOT SERIALIZED
+  , CA_LOCATION_TYPE                 VARCHAR(20) CHARACTER SET ISO88591 COLLATE
+      DEFAULT DEFAULT NULL NOT SERIALIZED
+  )
+  FOR HIVE.TPCDS_SF1000_ORC.customer_address1
+;
+
+--- SQL operation complete.
+```
+
++
+The size of the table _customer_address1_ is 71.1M.
++
+```
+[trafodion@nap001 ~]$ hdfs dfs -du -h /user/hive/warehouse/tpcds_sf1000_orc.db/customer_address1
+42.4 M  84.8 M  /user/hive/warehouse/tpcds_sf1000_orc.db/customer_address1/000000_0    
+28.7 M  57.4 M  /user/hive/warehouse/tpcds_sf1000_orc.db/customer_address1/000001_0
+```
+
++
+It takes approximately 13 seconds to get the result using the external table.
++
+```
+SQL>SELECT [ LAST 1 ] * FROM customer_address1 WHERE char_length(CA_ADDRESS_ID) > 100;        
+--- 0 row(s) selected. 
+
+Start Time             2018/01/31 02:21:55.409105
+End Time               2018/01/31 02:22:09.299215
+Elapsed Time                      00:00:13.890110
+Compile Time                      00:00:00.264101
+Execution Time                    00:00:13.625538
+```
+
+** This is the definition of the table _customer_address2_, it has same structure and size as the table _customer_address1_, but it does not have an external table.

```

Comment:
It might be good to talk about why performance is better when there is an external table. I can think of two reasons: 1. Trafodion statistics exist for the table. (Which seems to be the case in this example.) And 2. The Trafodion definition supplies upper bounds for varchar lengths, which may lead to better plans and/or run-time behavior. It is possible to have 2. without having 1. (As one can issue a CREATE EXTERNAL TABLE statement without also doing UPDATE STATISTICS.) So this is a key point: To get full performance benefit, one must also run UPDATE STATISTICS.